test/Queue.addN.spec.js

```diff
@@ -225,6 +228,19 @@ module.exports = function (queue) {
         });
     });
 
+    it('should return immediately on empty array', function () {
+      if (client) {
+        sinon.spy(client, 'send_command');
+      }
+      return queue.addN([])
+        .then(function (jobs) {
+          assert.deepEqual(jobs, []);
+          if (client) {
+            assert.strictEqual(client.send_command.calledOnce, false);
```

Comment:
I think this needs to be `called` and not `calledOnce`. What if it gets called twice?